lib/fog/kubevirt/compute/requests/get_server.rb

```diff
@@ -4,21 +4,43 @@ class Compute
       class Real
         def get_server(name)
           vm = get_raw_vm(name)
-          populate_runtime_info(vm)
+          vmi = runtime_vm(vm)
+
+          populate_runtime_info(vm, vmi)
           server = Server.parse vm
           populate_pvcs_for_vm(server)
+          populate_runtime_nets(vm, vmi, server)
           server
         end
 
         # Updates a given VM raw entity with vm instance info if exists
         #
         # @param vm [Hash] A hash with vm raw data.
-        def populate_runtime_info(vm)
-          vmi = get_vminstance(vm[:metadata][:name])
+        # @param vmi VMInstance object
+        def populate_runtime_info(vm, vmi)
+          return if vmi.nil?
+
           vm[:ip_address] = vmi[:ip_address]
           vm[:node_name] = vmi[:node_name]
           vm[:phase] = vmi[:status]
           vm
+        end
+
+        # Updates a given Server entity with vm instance networking details if exists
+        #
+        # @param vm [Hash] A hash with vm raw data.
+        # @param vmi VMInstance object
```

Comment:
just a nit, the server should be included in the parameter doc as well

Reply:
done